src/datatrove/pipeline/writers/disk_base.py

```diff
@@ -40,8 +40,13 @@ def __init__(
         self.compression = compression
         self.output_folder = get_datafolder(output_folder)
         output_filename = output_filename or self.default_output_filename
-        if self.compression == "gzip" and not output_filename.endswith(".gz"):
+        if output_filename.endswith(".parquet") and compression != "none":
```

Comment:
Or we can just use .parquet (not .{compression}.parquet) without adding here.
Then, we do not pass compression option at `src/datatrove/pipeline/writers/parquet.py` line 26.

change to 

```
super().__init__(
    output_folder,
    output_filename,
    compression=None,
    adapter=adapter,
    mode="wb",
    expand_metadata=expand_metadata,
    max_file_size=max_file_size,
)
```